app.js

```diff
@@ -0,0 +1,134 @@
+const Frame = require('./frame');
+const RollInput = require('./rollInput');
+
+class Application {
+  constructor(rollInput) {
+    this.rollInput = rollInput;
+    let frame1 = new Frame(1);
+    let frame2 = new Frame(2);
+    let frame3 = new Frame(3);
+    let frame4 = new Frame(4);
+    let frame5 = new Frame(5);
+    let frame6 = new Frame(6);
+    let frame7 = new Frame(7);
+    let frame8 = new Frame(8);
+    let frame9 = new Frame(9);
+    let frame10 = new Frame(10);
+    this.frames = [frame1, frame2, frame3, frame4, frame5, frame6, frame7, frame8, frame9, frame10];
+  };
+
+  getFrames() {
+    return this.frames;
+  };
+
+  printFrameNumber(frameNumber) {
+    console.log(`--- Frame ${frameNumber} ---`);
+  };
+
+  printStrike() {
+    console.log('STRIKE!');
+  }
+
+  printSpare() {
+    console.log('SPARE!');
+  }
```

Comment:
If you want to make the code a little bit cleaner you might want to separate methods/functions that interact with the user (or print something for the user) into a separate class, while keeping the score calculating methods in another.